add-ons/grafana-prometheus/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Add-on
+
+This addon installs grafana for metrics visualization from prometheus
+
+## Prerequisites
+
+The prometheus addon is required
+
+```
+minishift addon install prometheus-3.7
+minishift addon apply prometheus-3.7 --addon-env namespace=kube-system
```

Comment:
This command is wrong as it should be ` minishift addon install grafana-prometheus`. We need to pass the directory name to the `addon install` command.

Reply:
it's about installing the pre-requisite prometheus - not the grafana addon